packages/contracts-bedrock/README.md

```diff
@@ -89,3 +89,18 @@ to reduce the overhead of maintaining multiple ways to set up the state as well
 The L1 contract addresses are held in `deployments/hardhat/.deploy` and the L2 test state is held in a `.testdata` directory. The L1 addresses are used to create the L2 state
 and it is possible for stale addresses to be pulled into the L2 state, causing tests to fail. Stale addresses may happen if the order of the L1 deployments happen differently
 since some contracts are deployed using `CREATE`. Run `pnpm clean` and rerun the tests if they are failing for an unknown reason.
+
+### Static Analysis
+
+`contracts-bedrock` uses [slither](https://github.com/crytic/slither) as its primary static analysis tool. When opening a pr that includes changes to `contracts-bedrock`, you should
+verify that slither did not detect any new issues by running `pnpm slither:check`.
+
+If there are new issues, you should triage them.
+Run `pnpm slither:triage` to step through findings.
+You should _carefully_ walk through these findings, specifying which to triage/ignore (default is to keep all, outputting them into `slither-report.json`).
+Findings can be triaged into `slither.db.json` or kept in the `slither-report.json`.
+You should triage issues with extreme _care_ and security sign-off.
+
+After issues are triaged, or an updated slither report is generated, make sure to check in your changes to git.
+Once checked in, the changes can be verified by running `pnpm slither:check`.
+This will fail if there are issues missing from the `slither-report.json` that are _not_ triaged into `slither.db.json`.
```

Comment:
Do we need an issue to track performing the initial triage, since all findings are in `slither-report.json` which IIUC is a temporary measure until we have the DB? CI fails but seems to be marked as passed, and I'm unsure if that's related. 

<img width="1262" alt="image" src="https://github.com/ethereum-optimism/optimism/assets/17163988/e0dfba37-91b4-4d25-a137-b72758f06fd0">